### PR TITLE
feat(all-events) Add minidump all events

### PR DESCRIPTION
**Pull request by DominikB2014**
image

### CODE REVIEW

File: static/app/views/organizationGroupDetails/allEventsTable.tsx [hunk 1]
```diff
@@ -39,13 +39,18 @@ const AllEventsTable = (props: Props) => {
...
   ];
 
   const eventView: EventView = EventView.fromLocation(props.location);
   eventView.fields = fields.map(fieldName => ({field: fieldName}));
 
   eventView.sorts = decodeSorts(location).filter(sort => fields.includes(sort.field));
 
+  if (!eventView.sorts.length) {
+    eventView.sorts = [{field: 'timestamp', kind: 'desc'}];
+  }
```
Comment (DominikB2014) [inline static/app/views/organizationGroupDetails/allEventsTable.tsx:52]: Automatically sort by timestamp, if a supported sort isn't provided.

File: static/app/views/organizationGroupDetails/groupEvents.spec.jsx [hunk 2]
```diff
@@ -258,6 +256,57 @@ describe('groupEvents', function () {
       expect(attachmentsRequest).toHaveBeenCalled();
     });
 
+    it('does not display minidmp column with no minidumps', async () => {
```
Comment (udameli) [inline static/app/views/organizationGroupDetails/groupEvents.spec.jsx:259]:
```suggestion
    it('does not display minidump column with no minidumps', async () => {
```

File: static/app/utils/discover/fieldRenderers.tsx [hunk 3]
```diff
@@ -338,7 +348,30 @@ const SPECIAL_FIELDS: SpecialFields = {
...
+      );
+    },
+  },
+  minidump: {
+    sortField: null,
+    renderFunc: data => {
+      const attachments: Array<IssueAttachment & {url: string}> = data.attachments;
+
+      const minidump = attachments.find(
+        attachment => attachment.type === 'event.minidump'
```
Comment (udameli) [inline static/app/utils/discover/fieldRenderers.tsx:361]: could there be more than one minidump?

File: static/app/utils/discover/fieldRenderers.tsx [hunk 4]
```diff
@@ -338,7 +348,30 @@ const SPECIAL_FIELDS: SpecialFields = {
...
+      );
+
+      return (
+        <RightAlignedContainer>
+          <Button size="xs" disabled={!minidump ? true : false}>
```
Comment (udameli) [inline static/app/utils/discover/fieldRenderers.tsx:366]: nit: `!minidump` is already a boolean so you don't have to do a ternary here, just `disables={!minidump}` should suffice

File: static/app/utils/discover/fieldRenderers.tsx [hunk 5]
```diff
@@ -338,7 +348,30 @@ const SPECIAL_FIELDS: SpecialFields = {
...
+            <IconDownload
+              onClick={() => window.open(minidump?.url)}
+              color="gray500"
```
Comment (udameli) [inline static/app/utils/discover/fieldRenderers.tsx:369]: do we have that color in the theme? color as a variable would be easier to manage rather that a hardcoded one

File: static/app/utils/discover/fieldRenderers.tsx (hunk 4 above)
Comment (DominikB2014) [reply static/app/utils/discover/fieldRenderers.tsx:366]: Ohh right, I was using the string true and false for something else before. Anyways, will fix

File: static/app/views/organizationGroupDetails/groupEvents.spec.jsx (hunk 2 above)
Comment (DominikB2014) [reply static/app/views/organizationGroupDetails/groupEvents.spec.jsx]: fixed

File: static/app/utils/discover/fieldRenderers.tsx (hunk 3 above)
Comment (DominikB2014) [reply static/app/utils/discover/fieldRenderers.tsx:361]: Should be one minidump per event, that's how the current all events tab works as well

File: static/app/utils/discover/fieldRenderers.tsx (hunk 5 above)
Comment (udameli) [reply static/app/utils/discover/fieldRenderers.tsx:369]: this isn't critical but should be addressed soon

**Review by udameli — APPROVED** [review]

File: static/app/utils/discover/fieldRenderers.tsx (hunk 5 above)
Comment (DominikB2014) [reply static/app/utils/discover/fieldRenderers.tsx:369]: 👍 Yup, got a couple of cleanup items for this code like this don't change the logic, was planning on getting to it once the tab is verfied throughly in prod

Comment (DominikB2014) [reply static/app/utils/discover/fieldRenderers.tsx:369]: Fixed

**Pull request merged**